py/src/ai/h2o/sparkling/ml/algos/H2ODRF.py

```diff
@@ -0,0 +1,73 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+from pyspark import keyword_only
+
+from ai.h2o.sparkling import Initializer
+from ai.h2o.sparkling.ml.Utils import Utils
+from ai.h2o.sparkling.ml.algos.H2OAlgoBase import H2OAlgoBase
+from ai.h2o.sparkling.ml.params import H2ODRFParams
+
+javaMaxDoubleValue = (2 - 2 ** (-52)) * (2 ** 1023)
```

Comment:
WDYT about getting this separated to some utils class?

Reply:
👍 great idea